From c209305c3dd5e849a2b57e2e723474a2226e64c0 Mon Sep 17 00:00:00 2001
From: Rico Ganahl <rico.ganahl@bytesatwork.ch>
Date: Wed, 26 Oct 2022 09:06:45 +0200
Subject: [PATCH] configs: disable distro bootcmd on bytedevkit

Keep the u-boot environment simple by using good old boot command.

Signed-off-by: Rico Ganahl <rico.ganahl@bytesatwork.ch>
---
 configs/am62x_bytedevkit_a53_defconfig | 9 ++++++---
 1 file changed, 6 insertions(+), 3 deletions(-)

diff --git a/configs/am62x_bytedevkit_a53_defconfig b/configs/am62x_bytedevkit_a53_defconfig
index 0b5620afc4..b90e9de8f8 100644
--- a/configs/am62x_bytedevkit_a53_defconfig
+++ b/configs/am62x_bytedevkit_a53_defconfig
@@ -26,10 +26,11 @@ CONFIG_SPL_LIBDISK_SUPPORT=y
 CONFIG_SPL_SPI_FLASH_SUPPORT=y
 CONFIG_SPL_SPI_SUPPORT=y
 CONFIG_DEFAULT_DEVICE_TREE="k3-am625-bytedevkit"
-CONFIG_DISTRO_DEFAULTS=y
+CONFIG_ENV_VARS_UBOOT_CONFIG=y
 # CONFIG_SYS_MALLOC_CLEAR_ON_INIT is not set
 CONFIG_SPL_LOAD_FIT=y
 CONFIG_SPL_LOAD_FIT_ADDRESS=0x81000000
+CONFIG_SUPPORT_RAW_INITRD=y
 CONFIG_BOARD_LATE_INIT=y
 CONFIG_AVB_VERIFY=y
 CONFIG_ANDROID_AB=y
@@ -57,7 +58,7 @@ CONFIG_SPL_THERMAL=y
 CONFIG_SPL_USB_GADGET=y
 CONFIG_SPL_DFU=y
 CONFIG_SPL_YMODEM_SUPPORT=y
-# CONFIG_CMD_BOOTZ is not set
+CONFIG_HUSH_PARSER=y
 CONFIG_CMD_ADTIMG=y
 CONFIG_CMD_ABOOTIMG=y
 # CONFIG_CMD_EEPROM is not set
@@ -66,6 +67,8 @@ CONFIG_CMD_DM=y
 # CONFIG_CMD_GPIO is not set
 # CONFIG_CMD_SPI is not set
 CONFIG_CMD_AB_SELECT=y
+CONFIG_CMD_PXE=y
+CONFIG_CMD_SYSBOOT=y
 CONFIG_CMD_DDRSS=y
 # CONFIG_CMD_EXT4_WRITE is not set
 CONFIG_CMD_AVB=y
@@ -102,7 +105,6 @@ CONFIG_FASTBOOT_FLASH=y
 CONFIG_FASTBOOT_FLASH_MMC_DEV=0
 CONFIG_FASTBOOT_CMD_OEM_FORMAT=y
 CONFIG_TI_SCI_PROTOCOL=y
-CONFIG_DM_I2C=y
 CONFIG_SYS_I2C_OMAP24XX=y
 CONFIG_DM_MAILBOX=y
 CONFIG_K3_SEC_PROXY=y
@@ -161,6 +163,7 @@ CONFIG_SPL_DM_USB_GADGET=y
 CONFIG_USB_XHCI_HCD=y
 CONFIG_USB_DWC3=y
 CONFIG_USB_DWC3_GENERIC=y
+CONFIG_USB_STORAGE=y
 CONFIG_USB_GADGET=y
 CONFIG_USB_GADGET_MANUFACTURER="Texas Instruments"
 CONFIG_USB_GADGET_VENDOR_NUM=0x0451
-- 
2.39.2

